src/Core_Command.php

```diff
@@ -1042,53 +1070,52 @@ public function update( $args, $assoc_args ) {
 					list( $update ) = $from_api->updates;
 				}
 			}
-
-		} else if (	\WP_CLI\Utils\wp_version_compare( $assoc_args['version'], '<' )
+		} elseif ( \WP_CLI\Utils\wp_version_compare( $assoc_args['version'], '<' )
 			|| 'nightly' === $assoc_args['version']
 			|| \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' ) ) {
```

Comment:
You may want to make it slightly more easily detectable where the conditions end and the content of the control structure starts. (Here and possibly in other places too)
```suggestion
			|| \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' ) 
		) {
```